### DISCUSSION TITLE
Feat: Open Autocomplete window on cursor position change

### DISCUSSION THREAD

**Discussion by RyloRiz** [aws/codewhisperer-command-line-discussions#52] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
When using `marlonrichert/zsh-autocomplete` and hitting `Tab` to autocomplete the next suggestion, the CodeWhisperer suggestion window disappears until another character is typed. It would be a great help if the window appeared any time the cursor position changed (/current input changed) instead of when the user types a character themselves.

**Note: I have removed the default `Tab` keybind in CodeWhisperer that typically would have been bound to `Insert common prefix or shake`**
___

#### `zsh-autocomplete` suggesting a path & CodeWhisperer displaying window

<img width="394" alt="Screen Shot 2023-12-30 at 11 41 26 AM" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/91710711/5120901a-6da6-4ce6-8a7d-e42b08fdeb32">

#### `Tab` hit, `zsh-autocomplete` inserted path & CodeWhisperer no longer displaying window

<img width="274" alt="Screen Shot 2023-12-30 at 11 41 48 AM" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/91710711/02557cbe-0d63-4938-8fa3-6a67d15fbaf5">

**Comment by chaynabors** [comment]
This should already work as you expect. Can you share the output of `cw settings all` and @ me?

**Reply by RyloRiz** [reply]
<img width="392" alt="Screen Shot 2024-01-09 at 9 54 25 PM" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/91710711/930b685a-8017-478a-9cf5-0c2cdd9d05cd">

@chaynabors It could be due to this setting, but I want autocomplete to show automatically *and* when `Tab` is pressed. Additionally, this appears to not affect iTerm2 regardless of whether it's enabled or disabled.

<img width="628" alt="Screen Shot 2024-01-09 at 9 56 11 PM" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/91710711/cdb08b05-6da2-4c04-9040-dc171762e8bd">

**Comment by brendanfalk** [comment]
@RyloRiz We don't properly integrate with marlonrichert/zsh-autocomplete plugin. We have an open feature request for this integration to work properly here: https://github.com/aws/codewhisperer-command-line-discussions/discussions/20

I am going to close this issue and suggest you upvote the open feature request
